p-interface/src/lib.rs

```diff
@@ -8,3 +8,28 @@ pub mod state;
 pub mod program {
     pinocchio_pubkey::declare_id!("TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA");
 }
+
+
+#[cold]
+pub const fn unlikely_branch() {}
+
+
```

Comment:
/// Return the given `bool` value with a hint to the compiler that `true` is the
/// likely case.